### PR TITLE
Reasonable new event options

### PR DESCRIPTION
**Pull request by francescoannetta**
Implements #28 except for month specific day presentation. My preference would be to skip that one issue.

### CODE REVIEW

File: views/create-event.html [hunk 1]
```diff
@@ -14,107 +14,187 @@
...
-      <option value="17" >17</option>
-      <option value="18" >18</option>
-      <option value="19" >19</option>
-      <option value="20" >20</option>
-      <option value="21" >21</option>
-      <option value="22" >22</option>
-      <option value="23" >23</option>
+    <select name="hour" id="hour">
+      {% for hourNumber, hourName in allowedDateInfo.hours %}
+        <option value = "{{hourNumber}}">{{hourName}}</option></option>
```
Comment (dhatch) [inline views/create-event.html]: Two closing tags.

File: views/create-event.html [hunk 2]
```diff
@@ -14,107 +14,187 @@
...
+  </div>
     
+  <div class="form-group">
     <label for="minute">Event minute:</label>
-    <select name="minute" form="create-event">
-      <option value="00" selected>00</option> 
-      <option value="30" >30</option>
+    <select name="minute" id="minute">
+      {% for minute in allowedDateInfo.minutes %}
+        <option value = "{{minute}}">{{minute}}</option></option>
```
Comment (dhatch) [inline views/create-event.html]: Two closing tags.

File: views/create-event.html [hunk 3]
```diff
@@ -14,107 +14,187 @@
...
+      </select>
+    </div>
+-->
+
+<!--    
+    <div class="form-group">
+      <label for="minute">Event minute:</label>
+      <select name="minute" form="create-event">
+        <option value="00" selected>00</option> 
+        <option value="30" >30</option>
```
Comment (dhatch) [inline views/create-event.html]: Let's remove this commented out code before we merge.

**Comment by dhatch** [comment]
A few quick changes, otherwise looks good!

**Comment by francescoannetta** [comment]
@dhatch thanks for your keen eye. I think this is ready to go now.

**Pull request merged**